codespell_lib/data/dictionary.txt

```diff
@@ -28428,6 +28428,7 @@ pojrectors->projectors
 pojrects->projects
 poket->pocket
 polariy->polarity
+polciy->policy
```

Comment:
I would suggest adding
```
polcy->policy
polcies->policies
```

Reply:
Thanks a lot for the additional suggestions. These would extend the scope of this PR which items i haven't found / noticed on my own. If this is o.K. from your / project side i'm happy to apply them.

Reply:
It's certainly OK from my side. Your choice.

Reply:
Thanks a lot, updated the PR accordingly. :+1: